gradle.properties

```diff
@@ -1,3 +1,4 @@
+org.gradle.jvmargs=-Xmx1024m
```

Comment:
Isn't this the default? Is specifying it necessary?

Reply:
Somehow it was giving me OOM error when I didn't specify this...

Reply:
I was apparently wrong, or the posts online I found were wrong. Default is 512 MB apparently 

https://docs.gradle.org/current/userguide/build_environment.html#sec:configuring_jvm_memory